Hunting Queries/Corelight/CorelightCompressedFilesTransferredOverHTTP.yaml

```diff
@@ -0,0 +1,27 @@
+id: f37e076e-69f6-4fcf-81dc-ca2bc5929eb5
+name: Corelight - Compressed Files Transferred over HTTP
+description: |
+  'Query searches for top sources which transferred compressed archives over HTTP.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated